code/modules/projectiles/projectile/reusable/foam_dart.dm

```diff
@@ -21,14 +21,13 @@
 	newcasing.modified = modified
 	var/obj/projectile/bullet/reusable/foam_dart/newdart = newcasing.loaded_projectile
 	newdart.modified = modified
-	if(modified)
-		newdart.damage = 5
-		newdart.nodamage = FALSE
 	newdart.damage_type = damage_type
 	if(pen)
 		newdart.pen = pen
 		pen.forceMove(newdart)
 		pen = null
+		newdart.damage = 5
+		newdart.nodamage = FALSE
```

Comment:
```suggestion
		newdart.nodamage = FALSE
		newcasing.harmful = TRUE
```

Reply:
I presume this is for pacifism? But even so, It does not appear necessary, pacificists are unable to shoot recovered projectiles even without this addition in my testing